### DISCUSSION TITLE
Ultraloq Lock Is Not reporting locked after being unlocked via zwave.

### DISCUSSION THREAD

**Discussion by Daniel-dev22** [zwave-js/zwave-js#6710] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?

If I unlock the lock via home assistant lock entity it doesn't report locked when it's later locked manually or via auto lock. It only reports unlocked/locked when done at the lock.

What did you expect to happen instead?
Zwave/home assistant would receive notification of the lock being locked.

  1. Go to '...' lock entity
  2. Click on '...' unlock
  3. Scroll down to '...'

Anything else we should know?

Included using S2.

### Software versions

Driver (node-zwave-js): ...
zwave-js: 12.4.4
Z-Wave JS UI: ...
zwave-js-ui: 9.9.1.f6dff21
Home Assistant Z-Wave Integration: ...
Core 2023.3.1
Home Assistant Z-Wave JS Addon: ...
Using docker container for zwave js UI.
ioBroker.zwave2 Adapter: ...

If you are using something non-standard, tell us here: ...

### Device information

Manufacturer: ... ultraloq
Model name: ... U-BOLT-PRO-ZWAVE
Node ID: ... 22

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwave_js_lock_failed_to_report_unlocked.txt](https://github.com/zwave-js/node-zwave-js/files/14812023/zwave_js_lock_failed_to_report_unlocked.txt)

**Comment by AlCalzone** [comment, marked as answer]
This might be related to the issues discussed in https://github.com/zwave-js/node-zwave-js/issues/5445#issuecomment-1908780378

Please make sure your lock is on the latest firmware. If the issues persist, the manufacturer will have to do something about it - your log confirms that nothing lock-related is received after unlocking via Z-Wave.

**Reply by Daniel-dev22** [reply]
Yep it's on latest firmware I updated it via zwave when the firmware was released.

Firmware is 1.5.3

Unless there's something newer?

**Reply by AlCalzone** [reply]
Not that I'm aware of. At least there's nothing else available via the update service.

**Reply by Daniel-dev22** [reply]
> Not that I'm aware of. At least there's nothing else available via the update service.

It's just weird that I only noticed this recently it wasn't always the case. I can't say for sure when I started noticing this though.
